server/server.js

```diff
@@ -48,8 +61,9 @@ module.exports = function(port, middleware, callback) {
         var id  = req.params.id;
         var todo = getTodo(id);
         if (todo) {
-            if (req.body.title) {
+            if (typeof req.body.title !== "undefined" && typeof req.body.isComplete !== "undefined") {
```

Comment:
I think typeof is not required? if you're checking to see if the value is undefined.